lib/AST/ASTImporter.cpp

```diff
@@ -2321,13 +2321,14 @@ bool ASTNodeImporter::ImportTemplateInformation(FunctionDecl *FromFD,
 
 Decl *ASTNodeImporter::VisitFunctionDecl(FunctionDecl *D) {
 
-  // If we have the defintion in the ToCtx, import that and skip the prototype.
-  // This handles difficulties when we have a recursive function with a
-  // prototype and with a definition in the same ToCtx.  E.g. importing the
-  // prototype below would import the definition twice without this branch.
-  // "void f(); void f() { f(); }"
-  if (FunctionDecl* Definition = D->getDefinition())
-    D = Definition;
+//  FIXME: This part is commented out temporarily.
```

Comment:
Maybe adding a link to the closed pull request with the discussion might be useful in the future.